src/Features/Editor/ChangeLog/ChangeLogTable/ChangeLogTable.tsx

```diff
@@ -146,7 +146,7 @@ class ChangeLogTable extends Component<ChangeLogTableProps> {
             />
           </>
         ) : (
-          <Error404 header={null} message={null} title="No change logs found" />
+          <EmptyState header={null} message={null} title="No change logs found" />
```

Comment:
Depending on what you do for the defaults, one or two of these props aren't needed right?